src/core/client/admin/components/UserStatus/BanModal.tsx

```diff
@@ -55,8 +54,33 @@ interface Props {
   moderationScopesEnabled?: boolean | null;
   viewerScopes: Scopes;
   userRole: string;
+  isMultisite: boolean;
 }
 
+const createSingleSiteSubmitButton = (
+  isBanned: boolean,
+  lastFocusableRef: React.RefObject<any>,
+  disabled: boolean
+) => {
+  if (isBanned) {
+    return (
+      <Localized id="community-banModal-unban">
+        <Button type="submit" ref={lastFocusableRef} disabled={disabled}>
+          Unban
+        </Button>
+      </Localized>
+    );
+  } else {
+    return (
+      <Localized id="community-banModal-ban">
+        <Button type="submit" ref={lastFocusableRef} disabled={disabled}>
```

Comment:
Looking at the issue, this button should be changed to use the `alert` color option and be red in this case. Can you please confirm and update?